src/runtime_src/core/pcie/tools/xbmgmt/xmc.cpp

```diff
@@ -539,10 +570,11 @@ bool XMC_Flasher::isXMCReady()
     if (!xmcReady) {
         auto format = xrt_core::utils::ios_restore(std::cout);
         if (!mDev->get_sysfs_path("xmc", "").empty()) {
-            std::cout << "ERROR: XMC is not ready: 0x" << std::hex
-                << XMC_MODE() << std::endl;
+            std::cout << "ERROR: XMC is not ready: 0x" <<
```

Comment:
Probably `std::cerr`

Reply:
thanks, fixed.

Reply:
> thanks, fixed.

hmm, I will check with the team why use ios_restore. But this comment is irrelevant with this fix. 
We could fix/change this in the future when the requirement is clear.

Reply:
My wild guess is that the `ios_restore` was to counterbalance the `std::hex` but you have removed it...